pkg/api/types.go

```diff
@@ -554,7 +554,7 @@ type AgentPoolProfile struct {
 	MinCount                            *int                 `json:"minCount,omitempty"`
 	EnableAutoScaling                   *bool                `json:"enableAutoScaling,omitempty"`
 	AvailabilityZones                   []string             `json:"availabilityZones,omitempty"`
-	PlatformFaultDomainCount            *int                 `json:"platformFaultDomainCount,omitempty"`
+	PlatformFaultDomainCount            *int                 `json:"platformFaultDomainCount"`
```

Comment:
changed this to match MasterProfile : 
https://github.com/Azure/aks-engine/pull/2207/commits/7802a7af9118f48cfb4a460dd35e19b3d166c592#diff-9ff06ad723720f9428a65da3710cf436R481